### ISSUE TITLE
review of tutorials/basic_example.rst

### ISSUE THREAD

**Issue by dogoepp** [resibots/limbo#126]
This is for easier discussion of pull request #115.

# `tutorials/basic_example.rst`

## around line 1
What are the three cascading headers meaning ?

## around line 10
What about replacing "test" by something more specific, like "myExperiment" ?

## around line 93
> that we don't want to output any stats (by setting the dump period to -1)

I don't find this dump period, nor any -1 value in the example. Besides, I would interpret

```C++
struct bayes_opt_bobase {
  BO_PARAM(int, stats_enabled, true);
};
```

as telling to actually output the statistics.

**Comment by costashatz** [comment]
> What are the three cascading headers meaning ?

Probably nothing. I am removing them..

> What about replacing "test" by something more specific, like "myExperiment" ?

You are right. Will do.

> I don't find this dump period, nor any -1 value in the example

This was fix in #120 ...

**Comment by costashatz** [comment]
Fixed in c55be05. **Closing...**
